src/app/shared/pricing-definitions/pricing-definition/pricing-definition.component.ts

```diff
@@ -91,9 +91,9 @@ export class PricingDefinitionComponent implements OnInit {
   public restrictions!: FormGroup;
   // Duration
   public minDurationEnabled: AbstractControl;
-  public minDuration: AbstractControl;
+  public minDurationValue: AbstractControl;
```

Comment:
Because of the toggle() that takes the "this[`${event.source.id}Value`]" to set/unset the validators

Reply:
And will it work without the Value?
`this[${event.source.id}]`

Reply:
Came form the need to loop over controls enabled/values/units
I guess it can work without as we don't use the Unit part, Unit comes from i18n files

doing the refactor